public/app/plugins/datasource/mysql/configuration/ConfigurationEditor.tsx

```diff
@@ -150,6 +150,26 @@ export const ConfigurationEditor = (props: DataSourcePluginOptionsEditorProps<My
             value={jsonData.tlsSkipVerify || false}
           ></InlineSwitch>
         </InlineField>
+        <InlineField
+          labelWidth={WIDTH_MEDIUM}
+          tooltip={
+            <span>
+              Allows using the cleartext client side plugin if required by an account, such as one defined with the PAM
```

Comment:
I believe cleartext passwords were introduced in mysql 5.5.12; is it worth noting in the tooltip that it won't work for versions earlier than that?

Reply:
I think that whoever is configuring and enabling this is aware of that.